libsolidity/AST.h

```diff
@@ -440,13 +442,22 @@ class FunctionDefinition: public Declaration, public VariableScope, public Docum
 class VariableDeclaration: public Declaration
 {
 public:
-	VariableDeclaration(SourceLocation const& _location, ASTPointer<TypeName> const& _type,
-							ASTPointer<ASTString> const& _name, ASTPointer<Expression> _value,
-							Visibility _visibility,
-							bool _isStateVar = false, bool _isIndexed = false):
-			Declaration(_location, _name, _visibility),
-			m_typeName(_type), m_value(_value),
-			m_isStateVariable(_isStateVar), m_isIndexed(_isIndexed) {}
+	VariableDeclaration(
+			SourceLocation const& _location,
```

Comment:
I would say the initialization list should not be unindented. If you remember in the previews ci It was 1 tab for argument list and 2 for member initialization list but I agree that this way it looks better.
Should I change it or leave it like this guys?